package/src/components/TextInput/v1/TextInput.js

```diff
@@ -97,10 +96,9 @@ const StyledTextarea = Textarea.extend`
 
 const IconWrapper = styled.div`
   box-sizing: border-box;
-  color: ${applyValidationColor("inputIconColor")};
-  fill: currentColor;
```

Comment:
FYI - `fill` and `color` here will no longer customize the icon's color. Currently, the icon's color for TextInput will be set in the SVG itself. To modify the SVG color, you'd have to use a different SVG.

In ErrorsBlock, however, changing the ErrorsBlock's IconWrapper's `errorsBlockIconColor` variable _will_ allow users to customize the color.

The reason why it's a bit more complex to do this with TextInput's IconWrapper is because TextInput's IconWrapper supports multiple icons, and at least 1 of the more complex SVGs has more than 1 fill color. I made a ticket to make this more easily customizeable.

cc @aldeed